cpu/nrf52/include/periph_cpu.h

```diff
@@ -108,6 +113,49 @@ typedef struct {
 #define PERIPH_I2C_NEED_WRITE_REG
 /** @} */
 
+/**
+ * @name    The PWM unit on the nRF52 supports 4 channels per device
+ */
+#define PWM_CHANNELS        (4U)
+
+/**
+ * @name    Generate PWM mode values
+ *
+ * To encode the PWM mode, we use two bit:
+ * - bit  0: select up or up-and-down counting
+ * - bit 15: select polarity
+ */
+#define PWM_MODE(ud, pol)   (ud | (pol << 15))
+
+/**
+ * @brief   Override the PWM mode definitions
+ * @{
+ */
+#define HAVE_PWM_MODE_T
+typedef enum {
+    PWM_LEFT       = PWM_MODE(0, 1),    /**< left aligned PWM */
+    PWM_RIGHT      = PWM_MODE(0, 0),    /**< right aligned PWM */
+    PWM_CENTER     = PWM_MODE(1, 1),    /**< not supported */
```

Comment:
Should we remove this?

Reply:
I don't have a strong opinion about that. As i saw in other cpus (samd21,efm32) center mode is not supported but PWM_CENTER is set anyway. I did not question this design, but i think it gives easy access / guidance to anyone who wants to implement center mode in the future.

Reply:
Fine I leave it up to you. Please squash